src/status_im/ui/screens/chat/views.cljs

```diff
@@ -334,6 +344,7 @@
                                           :idx                idx
                                           :list-ref           messages-list-ref}])
            :inverted                  true
+           :onViewableItemsChanged    on-viewable-items-changed
```

Comment:
are the default good enough?
there is a bunch of visibility config for this for instance

```
:viewability-config (clj->js {:waitForInteraction false
                                                           :minimumViewTime 100
                                                           :viewAreaCoveragePercentThreshold 60})
```

Reply:
They should be ok